friends/src/Components/FriendsList.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const List = styled.div`
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+`;
+const Card = styled.div`
+background-color: lightblue;
+width: 200px;
+margin: 30px 0;
+`;
+
+class FriendsList extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            friends: []
+        };
```

Comment:
I would recommend moving up the state variable for the `friends` object array to `App.js` so that the method used for adding new friends in the `FriendsForm` component can also access the array and add more friend objects.